generators/server/templates/src/main/java/package/security/jwt/_TokenProvider.java

```diff
@@ -90,7 +90,15 @@ public boolean validateToken(String authToken) {
             return true;
         } catch (SignatureException e) {
             log.info("Invalid JWT signature: " + e.getMessage());
```

Comment:
I just kept the initial logging. But, definitely, I can log the exception as requested